uhyve-interface/Cargo.toml

```
@@ -0,0 +1,17 @@
+[package]
+name = "uhyve-interface"
+version = "0.1.0"
+edition = "2021"
+authors = [
+  "Jonathan Klimt <jonathan.klimt@eonerc.rwth-aachen.de>",
+  "Stefan Lankes <slankes@eonerc.rwth-aachen.de>",
+]
+description = "The interface between uhyve and a guest VM"
+repository = "https://github.com/hermitcore/uhyve"
+license = "MIT/Apache-2.0"
```

See https://github.com/hermitcore/uhyve/pull/459

```suggestion
license = "MIT OR Apache-2.0"
```